src/components/Modal/Modal.module.css

```diff
@@ -34,7 +34,7 @@
   justify-content: center;
   padding: var(--eds-size-2);
 
-  z-index: var(--eds-z-index-top); /* 1 */
+  z-index: 1050; /* 1 */
```

Comment:
can you leave a comment explaining why this doesn't use a design token for now?

Reply:
Yes, will do!